arangodb-net-standard/SerializerException.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace ArangoDBNetStandard
```

Comment:
Maybe put this file and class in the `Serialization` folder/namespace? It would make sense to me to group them with other serialization-related classes.